unittest/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ target_include_directories(${TARGET_NAME}
     ${NAPA_ROOT}/inc
     ${NAPA_ROOT}/src
     ${NAPA_ROOT}/third-party
-    "/usr/local/include/node")
+    ${CMAKE_JS_INC})
```

Comment:
please take a look of the unittest case of "scheduler-tests". As was mentioned to you about the problem, it fails because it depends on ~scheduler() to force draining tasks and release workers after that, which is not the way that new napa-vnext uses